client/lib/directly/index.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+// import { loadScript } from 'lib/load-script';
+
+const DIRECTLY_SCRIPT_PATH = 'https://www.directly.com/widgets/rtm/embed.js';
+const DEFAULT_RTM_WIDGET_OPTIONS = {
+	// Automattic widget ID
+	id: '8a2968fc57d1e2f40157f42bf2d43160',
+	// By default don't show the "Ask a Question" button
+	displayAskQuestion: false
+};
+
+let initialized = false;
+
+const loadDirectlyScript = ( callback = noop ) => {
```

Comment:
> I'm also curious how this affects the larger conversation happening now about performance in Calypso? 

So like you said this should be handled by code splitting as long as it is `required` when necessary. I'd give https://github.com/Automattic/wp-calypso/tree/master/server/bundler/babel/babel-plugin-transform-wpcalypso-async a read. the new `asyncRequire` transform might be what your looking for here.

> a nicety of linking to the remote script is we can do nothing and receive changes (bugfixes, new features, etc) from Directly

Not really a nicety from my point of view since I've been burned before by updates that break things unexpectedly. I would like for Directly to update and keep things 100% backwards compatible but I don't expect it.

> My instinct is that the safest thing is to model our implementation in the spirit of the vendor's instructions as much as possible, but if that's not standard procedures here that's 👌 with me.

I'm with you on that if our implementation looks similar to theirs :smile:. I keep going back to my previous mediocre _(not bad but also not great)_ relationship with olark but the problem we've had with their API is that it wasn't built with the expectation that it'd run from within a single page load javascript application. That means if we wan't to use it with calypso in a way thats meaningful we need to massage it into what we need. I haven't looked at Directly's stuff as much as you have so I don't know 100% how it works so my advice is to use your best judgement there. If you think pulling down the script locally is a recipe for disaster then lets not do it but lets weigh the benefits either way :smile:

> Sorry for so many questions, but I'm trying to make sure I understand the Calypso way of doing things 

Not a problem. There aren't many third party integrations within calypso so this is kinda still new territory